README.md

```diff
@@ -31,10 +31,16 @@ Usage: staledeps <path(s)> [options]
 Options:
       --version    Show version number                                 [boolean]
   -h, --help       Show help                                           [boolean]
+  -d, --sort-dir   Direction to sort data
+                              [string] [choices: "asc", "desc"] [default: "asc"]
```

Comment:
I've just used the exact output from `--help` from yargs. I could make the change, but I'd suggest we just keep the exact output as it's easier to maintain and more consistent, instead of trying to tweak it ourselves.